src/common/enums/sourceTypes.ts

```diff
@@ -1,4 +1,5 @@
 export enum SourceTypes {
   GPKG = 'geopackage',
   S3 = 's3',
+  FS = 'file',
```

Comment:
its 'file' - because mapproxy defined this value as file system "cache dir" inside the yaml file